.github/workflows/tag-publish.yml

```diff
@@ -44,6 +44,10 @@ env:
   GOOGLE_PROJECT: broad-dsp-gcr-public
 
 jobs:
+  tag:
+    uses: ./.github/workflows/tag.yml
```

Comment:
Will this always bump the version? I don't see any conditional logic in the `tag` workflow

Reply:
Good question. I assumed that since tag is triggered by publish workflow(which has conditional logic) this should be invoked conditionally too.

Reply:
I decided to simplify and just upgrade the version of bumper action we use. I see the same pattern in WSM.